src/dsmr/reader.h

```diff
@@ -56,8 +56,8 @@ namespace dsmr {
  *  - parse() is called
  *  - loop() is called and the start of a new message is available
  *
- * When disable is called, the request pin is disabled again and any
- * partial message is discarded. Any bytes received while disabled are
+ * When disable is called, the request pin is DISABLED_STATE again and any
+ * partial message is discarded. Any bytes received while DISABLED_STATE are
```

Comment:
Seems you had a overly zealous search-replace here (and in other places in this commit).